cleverhans/model.py

```diff
@@ -1,4 +1,5 @@
 from abc import ABCMeta
+import tensorflow as tf
```

Comment:
New import but not used.

Reply:
True, updated PR will reflect the change.